app/src/main/java/org/mozilla/fenix/settings/DropDownListPreference.kt

```diff
@@ -23,3 +24,13 @@ class DropDownListPreference @JvmOverloads constructor(
         return ArrayAdapter(context, R.layout.etp_dropdown_item)
     }
 }
+
+fun DropDownListPreference.findEntriesValue(newValue: Any?): CharSequence? {
+    val newValueString = newValue as? String ?: return null
+    val index = this.findIndexOfValue(newValueString)
```

Comment:
So why when we get the EntriesValue do we have to find the index of value of this valueString and then return the entry based on the index? What is the `newValueString` then?

Reply:
We have two arrays that store entries and entries values. Previously we received `social` for example, but we needed `Cross-site and social media trackers`, otherwise the preference would not have updated.